tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -16,6 +16,7 @@
 #include <map>

```

Comment:
What is it that I added that needs `usual_latest_matrix.lst` instead of `concepts_matrix.lst` (or should it have always been  using the usual_latest_matrix?)

Reply:
There is currently _no_ `env.lst` for this test. I suggest `usual_latest_matrix` because this is C++20-only code (latest) which IIRC  doesn't use concepts/*requires-clause*s so it doesn't need `concepts_matrix`.

Reply:
https://github.com/microsoft/STL/blob/feature/spaceship/tests/std/tests/P1614R2_spaceship/env.lst

Reply:
🤦‍♂️ How'd I convince myself this was an entirely new test??!? Sorry.